onboarding-checklist.md

```diff
@@ -6,6 +6,9 @@ This checklist serves as a guide for the steps being taken when onboarding new p
 * Plan meeting with maintainers to discuss onboarding and clear expectations, and plan next steps
 * Add project representative to Technical Committee
 * Move repositories to OpenRail Association organization on GitHub
+  * License review
+  * Move the repos
+  * Set up teams/people/collaborators with access to the repos
 * Move web site to GitHub Pages within the OpenRail Association organization (if separate web page exists)
```

Comment:
According to the principles, also the domains have to be moved to Association, right?